chart/values.yaml

```diff
@@ -0,0 +1,140 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Default values for banyandb.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+standalone:
+  name: banyandb
+  podAnnotations: 
+    example: banyandb-foo
+  securityContext: {}
+    # runAsUser: 1000
+    # runAsGroup: 1000
+    # fsGroup: 1000
+  config: ["standalone"]
+  image:
+    repository: ghcr.io/apache/skywalking-banyandb
+    tag: 7443bd36e56404ee813b66a5e2b183d3c9ed3371
+    pullPolicy: IfNotPresent
+  env: []
+    # - name: BANYANDB_FOO
+    #   value: "bar"
+  priorityClassName: ""
+  podDisruptionBudget: {}
+    # minAvailable: 1
+    # maxUnavailable: 2
+    # matchLabels:
+    #   - key: foo
+    #     value: bar
+    # matchExpressions:
+    #   - key: foo
+    #     operator: In
+    #     values: [bar, baz]
+    # paused: false
+  tolerations: []
+    # - key: foo
+    #   value: bar
+    #   operator: Equal
+    #   effect: NoSchedule
+  nodeSelector: []
+    # - key: foo
+    #   value: bar
+  affinity: {}
+    # nodeAffinity:
+    #   requiredDuringSchedulingIgnoredDuringExecution:
+    #     nodeSelectorTerms:
+    #       - matchExpressions:
+    #           - key: foo
+    #             operator: In
+    #             values:
+    #               - bar
+    #               - baz
+    # podAffinity:
+    #   requiredDuringSchedulingIgnoredDuringExecution:
+    #     - labelSelector:
+    #         matchLabels:
+    #           - key: app
+    #             value: banyandb
+    #       topologyKey: "kubernetes.io/hostname"
+    #       namespaces: []
+  resources:
+    requests: []
+      # - key: cpu
+      #   value: "100m"
+      # - key: memory
+      #   value: "256Mi"
+    limits: []
+      # - key: cpu
+      #   value: "500m"
+      # - key: memory
+      #   value: "512Mi"
+
+  grpcSvc:
```

Comment:
We should support TLS settings.

1. For bringing your certification. This chart has to support load secrets by their name. 
2. For external certificate authorities, for instance Vault. The pod annotation can handle pk injection. But it would help if you documented where the banyandb server loads the pk and certs and what their names are.